### ISSUE TITLE
[annotation-toolkit] `Layer.getData()` doesn't always work, needs workaround `Layer.onWithGroup = true`

### ISSUE THREAD

**Issue by pringshia** [facebookresearch/Mephisto#430]
For some reason, this doesn't work:

```jsx
              <Layer
                displayName="Layer 1"
                icon="layer"
                getData={({ store }) => {
                  log(store.get("data")); /* This never gets executed! */
                  return { data: store.get("data") };
                }}
                component={({ data }) => {
                  return (
                    <div>
                      Data: {JSON.stringify(data)}
                    </div>
                  );
                }}
              />

```

https://codesandbox.io/s/annotation-toolkit-embed-test-nlmub?file=/src/AnnotationTool.js:1438-1576

Adding `onWithGroup={true}` however makes things work as expected. Could be something to do with how the Layers are being rendered with ContentPanel.js.

**Comment by pringshia** [comment]
I suspect this is the issue.

Secondary activations spread in `getData()`:

https://github.com/facebookresearch/Mephisto/blob/f489992bf76d2b048b42395096267a2c1bf34738/packages/annotation-toolkit/src/panels/ContentPanel.js#L102

However primary activations are missing the `getData()` spread:

https://github.com/facebookresearch/Mephisto/blob/f489992bf76d2b048b42395096267a2c1bf34738/packages/annotation-toolkit/src/panels/ContentPanel.js#L89

**Comment by pringshia** [comment]
Fixed with annotation-toolkit@1.0.5
